src/main.ts

```diff
@@ -108,12 +119,41 @@ export async function run(inputs: RunInputs): Promise<void> {
   }
 }
 
+const createSingleSubjectAttestations = async (
+  subjects: Subject[],
+  predicate: Predicate,
+  opts: CreateAttestationOptions
+): Promise<AttestResult[]> => {
+  const atts: AttestResult[] = []
+  // Generate one attestation for each subject
+  for (const subject of subjects) {
+    const att = await createAttestation([subject], predicate, opts)
+    atts.push(att)
+  }
+  return atts
+}
+
+const createMultiSubjectAttestation = async (
+  subjects: Subject[],
+  predicate: Predicate,
+  opts: CreateAttestationOptions
+): Promise<AttestResult[]> => {
+  const att = await createAttestation(subjects, predicate, opts)
+
+  if (att.attestationID) {
+    core.setOutput('attestation-id', att.attestationID)
+    core.setOutput('attestation-url', attestationURL(att.attestationID))
+  }
```

Comment:
Setting these outputs only makes sense when we generate a single attestation.

Reply:
Let's print these out regardless for consistency / debugging - I think it's OK if we just print the the id/url for the first attestation.

This goes hand in hand with unwinding it into the main func per my comment above (i.e. `att[0].attestationID`)

Reply:
Done, changed to always set the outputs using `atts[0]`.